modules/bq-log-alerting/main.tf

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#-----------------#
+# Local variables #
+#-----------------#
+locals {
+  actual_source_name = var.source_name != "" ? var.source_name : google_scc_source.bq_log_alerts[0].name
+}
+
+#----------------#
+# API activation #
+#----------------#
+resource "google_project_service" "enable_bigquery_api" {
```

Comment:
Enabling APIs are usually out scope for a module. This can be just listed as a pre req https://github.com/terraform-google-modules/terraform-google-log-export/blob/c2e59c9993977de4364a9baacfef601b6b587c5e/modules/bq-log-alerting/README.md#apis